domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapperTest.java

```diff
@@ -1010,18 +1027,49 @@ private FieldType mapAssertCommonFieldsAndReturn(FieldTypeEntity fieldTypeEntity
             fieldTypeEntity.addListItems(
                 asList(fieldTypeListItemEntity1, fieldTypeListItemEntity2, fieldTypeListItemEntity3));
 
+            FieldTypeEntity collectionFieldTypeEntity = fieldTypeEntity("CollectionFieldType");
+            FieldType collectionFieldType = new FieldType();
+            when(spyOnClassUnderTest.map(collectionFieldTypeEntity)).thenReturn(collectionFieldType);
+
+            FieldType fieldType = spyOnClassUnderTest.map(fieldTypeEntity);
+
+            assertEquals(fieldTypeEntity.getReference(), fieldType.getId());
+            assertEquals(fieldTypeEntity.getMinimum(), fieldType.getMin());
+            assertEquals(fieldTypeEntity.getMaximum(), fieldType.getMax());
+            assertEquals(fieldTypeEntity.getRegularExpression(), fieldType.getRegularExpression());
+
+            assertEquals(fieldTypeEntity.getListItems().size(), fieldType.getFixedListItems().size());
+            assertEquals(fieldType.getFixedListItems().get(0).getLabel(), fixedListItem1.getLabel());
+            assertEquals(fieldType.getFixedListItems().get(1).getLabel(), fixedListItem2.getLabel());
+            assertEquals(fieldType.getFixedListItems().get(2).getLabel(), fixedListItem3.getLabel());
+
+            assertEquals(fieldTypeEntity.getCollectionFieldType(), fieldType.getCollectionFieldType());
+
+            return fieldType;
+
```

Comment:
should we add a test for verifying that the INSTANCE mapper not being called for non-complex entity types?
I mean sth like `verify(spyOnClassUnderTest, never()).map(fieldTypeEntity)`